lib/web_ids.js

```diff
@@ -95,12 +95,12 @@ function checkIp(srcIp) {
 
   IP_SIGNATURES.forEach((ip) => {
     if(srcIp.match(ip)){
-
-     
+      
       const type = 'Warning - potential intrusion';
       const message = 'Known malicious ip: ' + scrIP + 'detected.';
   
       appendToLog(LOG_FILE, type, message);
+      break;
```

Comment:
Unfortunately this doesn't work in javascript `forEach` loop. I will adjust it to `for...of...` loop so that we can break out of it.

Reply:
Oh okay sorry did not know that, thanks!